fairing/preprocessors/full_notebook.py

```diff
@@ -7,16 +7,21 @@
 class FullNotebookPreProcessor(BasePreProcessor):
     def __init__(self,
                  notebook_file=None,
-                 command=["jupyter", "nbconvert", "--stdout", "--to", "notebook", "--execute"],
+                 input_files=None,
+                 command=["jupyter", "nbconvert", "--stdout", "--to", "notebook", "--execute", "--allow-errors",
```

Comment:
I think users should be able to configure allow errors behavior. I don't see executing past an error is that common of a use case but on the other hand users may not want to run costly operations like training if any preprocessing is failed.

Reply:
Added a todo for now. From testing, I've found that nbconvert does not display any output while the notebook is executing, which makes it challenging to monitor long-running jobs.

So we will want to prioritize Papermill (#33) and we can add additional configuration options there.